packages/next/src/server/stream-utils/encode-decode.ts

```diff
@@ -4,7 +4,8 @@ export function encodeText(input: string) {
 
 export function decodeText(
   input: Uint8Array | undefined,
-  textDecoder: TextDecoder
+  textDecoder: TextDecoder,
+  isStream = true
```

Comment:
How about use explict arg instead of opional so make sure we can cover all cases